### ISSUE TITLE
Write the rules for Demo Day

### ISSUE THREAD

**Issue by 149203** [149203/developersvegas2#3]
We want short, simple rules that sum up our expectations for Demo Day participants.

**Comment by yasso1am** [comment]
I'm going to dig into this. My goal is to be as succinct as possible, but I'm wondering if you have a number of rules in your head that you think people would be willing to read?

**Comment by yasso1am** [comment]
Additionally, can I do this in markdown? I've never used markdown and this may be an opportunity for me to practice.

**Comment by 149203** [comment]
If you post the rules right here, it will be in markdown. So just do that.

4-16 rules.

**Comment by yasso1am** [comment]
Isn't that cheating? I'm just using the rich-text-editor/gui of github text box to create my markdown?

**Comment by 149203** [comment]
# Type it instead of using the GUI

## There's not really much to markdown. That's why it's _markdown_ and not _markup_.

**Comment by yasso1am** [comment]
Ah, I see. What you're saying is I can write in markdown in this box and it will look as desired. When/if you copy paste it will already be in markdown. Got it.

**Comment by yasso1am** [comment]
_This should be in italics_
## This should be bold
# This should be more bolder

**Comment by 149203** [comment]
https://guides.github.com/features/mastering-markdown/

**Comment by 149203** [comment]
@yasso1am I updated the old site here:

http://developers.vegas/

I've also added you as a collaborator on that private repo. If you see any changes you'd like to make, submit a pull request.

Two sentiments I wasn't able to express in the FAQ, but maybe they're better as internal values:

1. Kind > Helpful > Knowledgeable

The `>` means "greater than". While all three are desired, it's better to be kind than helpful, and better to be helpful than knowledgeable.

2. Las Vegas Developers > Our sponsors > the Las Vegas tech ecosystem.

We prioritize our developer community, then our sponsors, then our Las Vegas ecosystem.
